app/src/main/java/com/eveningoutpost/dexdrip/Models/blueReader.java

```diff
@@ -1,20 +1,40 @@
 package com.eveningoutpost.dexdrip.Models;
+
+import android.text.format.DateFormat;
 import com.eveningoutpost.dexdrip.Home;
 import java.nio.ByteBuffer;
 import java.util.Date;
 import com.eveningoutpost.dexdrip.Services.DexCollectionService;
 import com.eveningoutpost.dexdrip.Models.UserError.Log;
+import com.eveningoutpost.dexdrip.UtilityModels.Notifications;
 import com.eveningoutpost.dexdrip.UtilityModels.PersistentStore;
 import com.eveningoutpost.dexdrip.UtilityModels.Pref;
 import com.eveningoutpost.dexdrip.xdrip;
+import com.eveningoutpost.dexdrip.R;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import static com.eveningoutpost.dexdrip.utils.FileUtils.getExternalDir;
+import static com.eveningoutpost.dexdrip.utils.FileUtils.makeSureDirectoryExists;
+
 
 /**
  * Created by MasterPlexus on 11.12.2017.
  */
 
 public class blueReader {
     private static final String TAG = "blueReader";
+    private static final String BatLog="/BatteryLog.csv";
     private static int counterHibernated = 0;
+    private static String tempVers="";
+
+    private final static byte[] shutdown = new byte[]{0x6B};
```

Comment:
Please add a comment about what therse bytes are converted to letters. This makes it a bit easier to understand and compare to the docs supplied by blueReader on how do communicate.